Automattic/wp-calypso
# Remove Jetpack Pricing Asterisks

#### Changes proposed in this Pull Request

* Remove all asterisks from Jetpack Pricing Pages
* Remove asterisk footnotes

| Jetpack Pricing Page (Top) | Jetpack Pricing Page (Bottom) | Storage Upgrade Page |
| -------------------------- | ----------------------------- | -------------------- |
| <img width="964" alt="Screen Shot 2022-02-15 at 09 04 19" src="https://user-images.githubusercontent.com/2810519/154112532-ff1f6999-d31b-40d8-80b9-921265850811.png"> | <img width="927" alt="Screen Shot 2022-02-15 at 09 04 49" src="https://user-images.githubusercontent.com/2810519/154112582-92394002-6629-4398-add9-7880d997ffa0.png"> | <img width="599" alt="Screen Shot 2022-02-15 at 09 03 43" src="https://user-images.githubusercontent.com/2810519/154112651-2a39692c-48bd-4fd2-a93a-faed37b40c99.png"> |


#### Testing instructions

1. Boot Branch or use Live Link as Calypso Green.
2. Navigate to `/pricing`.
3. Verify that the following asterisks have been removed:
   - The Intro Pricing Banner
   - The Discount Labels on each Pricing Card
4. Verify the footnotes at the bottom of the page ( below the FAQ ) is gone
5. Boot Branch or use Live Link as Calypso Blue.
6. Navigate to `/plans/storage/:siteId` for a site with Backup T1 or Security T1.
7. Verify that the Asterisks on the "save 50% off" monthly/annual toggle is gone.
8. Verify the footnotes at the bottom of the page ( below the FAQ ) are gone.

Related to 1196108640073826-as-1201681966634228
